### PR TITLE
[[ Bug 20928 ]] MultipleHilites is MultipleLines where it should not

### PR DESCRIPTION
**Pull request by livecodemiriam**

### CODE REVIEW

**Comment by jameshale** [comment]
wow! this "error" has been in the docs forever (well at least LC 4.5.3 which is the oldest I have on disk).
Although the entry for "multipleLines" clearly states it is a synonym of "multipleHilites" the entry for both it and "multipleHilites" only speaks in terms of "multipleLines".
Well caught.

File: docs/notes/bugfix-20928.md [hunk 1]
```diff
@@ -0,0 +1 @@
+[[ Bug 20928 ]] MultipleHilites is MultipleLines where it should not
```
Comment (livecodeali) [inline docs/notes/bugfix-20928.md]: The content of this file should be something like
```
# Make multipleHilites the primary entry for the multipleLines property
```

**Comment by livecodeali** [comment]
@livecode-vulcan review ok a858c65

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok a858c650186f7f8228922fdfbfc8ee892eb4e691

**Comment by livecode-vulcan** [comment]
😎 test success a858c650186f7f8228922fdfbfc8ee892eb4e691

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/1244)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/1260)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/1234)
* try-community-universal-ios-iphoneos11.2: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos11.2/builds/187)
* try-community-universal-ios-iphonesimulator11.2: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator11.2/builds/186)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/1238)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1264)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/1244)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1260)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/1243)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/1253)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/1296)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/1236)

**Pull request merged**